src/global.styles.scss

```diff
@@ -41,3 +41,14 @@ $bold: 800; // Bold
 $border-radius: 0.3rem;
 $element-box-shadow-light: 0 5px 10px -5px #ccc, 0 -1px 10px -5px #eee;
 $element-box-shadow-dark: 0 5px 10px -5px #222, 0 -1px 10px -5px #222;
+
+body {
+    box-sizing: border-box;
+    margin: 20px;
+    width: 100%;
```

Comment:
Putting margin on the body element lowers the header as well and on the sided it is not needed and on the bottom it should be styled on the element that is last in the page - margin not needed here.
why did you set width 100% here?